news/10020.bugfix.rst

```diff
@@ -0,0 +1 @@
+The presence of an old version of Tornado (before 4.1) in the environment no longer breaks pip. 
```

Comment:
Describe the problem in past tense, and the fix in present tense (because that's what makes sense after the release is made). So maybe something like

> Remove unused optional `tornado` import in vendored `tenacity` to avoid an old version or `tornado` installation from breaking pip.